src/IPub/FormDateTime/Controls/Date.php

```diff
@@ -369,7 +369,7 @@ protected function getControlSettings()
 				// Day of the week start. 0 (Sunday) to 6 (Saturday)
 				'weekStart'				=> $this->weekStart,
 				// Whether or not to close the date picker immediately when a date is selected
-				'autoclose'				=> $this->autoclose ? 'true' : 'false',
+				'autoclose'				=> (bool)$this->autoclose,
```

Comment:
please add space between data type and property (on all changed rows)

Reply:
done